templates/namespace/namespace.py

```diff
@@ -38,11 +38,20 @@ def _monitor(self):
         except StateCheckError:
             return
 
+        def _reinstall():
+            self.state.delete('status', 'running')
+            self.install()
+
+        if not self.data['zerodb']:
```

Comment:
@zaibon I added this `if` for cases where the reinstall at line 54 fails, so next time it is monitoring it would immediately install again since it is not linked to a zerodb anymore.